baseplate/experiments/providers/simple_experiment.py

```diff
@@ -27,6 +27,55 @@
 }
 
 
+def _generate_overrides(override_config):
+    """Generate a dictionary of overrides.
+
+    The format of the override config is expected to be a list of dicts,
+    where each dict contains a single key (the treatment to assign) and value
+    (the targeting tree for that treatment).
+
+    This combination is used to ensure that the ordering of the overrides is
+    maintained, allowing consistent overriding for any overlapping area
+    across targeting trees. Each incoming variant request will check each
+    targeting override in sequence until one is matched or none match.
+
+    If the override configuration does not contain a wrapping list (ie, it
+    is just a dictionary), then overrides will work, but override ordering
+    is not guaranteed. This means that if there is overlap between overrides
+    (ie, a particular user is eligible for two overrides), then the results
+    may not be consistent.
+
+    In the event of an invalid override:
+    1) if a dict of variant : target is not provided, that list item will be
+        ignored. All other overrides will still remain active.
+    2) if the targeting object is invalid, the override will remain active,
+        but will return false for all targeting
+
+    Multiple targets can be provided for a single variant.
+    """
+
+    override_list = []
+    if override_config and isinstance(override_config, dict):
+        override_config = [override_config]
+
+    if override_config and isinstance(override_config, list):
```

Comment:
You might want to log a warning if you don't enter this if block since that means someone is giving you an invalid config.

Reply:
Good catch. Will do.